schedview/collect/nightreport.py

```diff
@@ -0,0 +1,120 @@
+import datetime
+from typing import Literal
+
+import requests
+
+from schedview.dayobs import DayObs
+
+MAX_RETRIES = 1
```

Comment:
MAX_RETRIES should be 2, based on my experience with the logging endpoints.

Reply:
Done